app/views/conferences/_booths.html.haml

```diff
@@ -0,0 +1,26 @@
+= content_for :splash_nav do
+  %li
+    %a.smoothscroll{ href: '#booths' } Booths
+
+.container
+  .row
+    .col-md-12.text-center
+      %h2 Booths
+  - i = 0;
+  - @conference.booths.confirmed.each_slice(3) do |slice|
+    .row.row-centered
+      - slice.each do |booth|
+        .col-md-4.col-sm-4.col-centered.col-top
+          .thumbnail
+            - if booth.logo_link
+              = link_to booth.website_url, class: 'thumbnail' do
+                = image_tag booth.picture.large.url
+            .caption
+              %h3.text-center
+                = booth.title
+              %p.text-center.text-muted
+                = link_to "#show_descrition_#{i}", "data-toggle"=>"collapse" do
+                  learn more
+              .collapse{ id: "show_descrition_#{i}" }
+                = markdown(booth.description)
+        - i = i + 1
```

Comment:
@differentreality  it increases the value of i by one. We need to do this outside of the `.slice(3)` otherwise `i` will take the value of 0-2. For example, if you have 6 confirmed booths, when you click  the 'learn more' of the booth number 5 (middle booth of the second row) it would trigger the collapsing of the booth with id `show_descrition_1` and not the one we would want ( `show_descrition_4` ). It was a bug that I hadn't notice before.